packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -304,14 +310,11 @@ describe('account', () => {
             const source = /* GraphQL */ `
                 query testQuery($address: String!) {
                     account(address: $address, encoding: PARSED) {
```

Comment:
So, here's how I think about this.

People should just straight up not be bothered with these fetch modes. It should just never surface in the GraphQL API.

I should absolutely not give a shit about specifying an encoding, I should just do this.

```graphql
account(address: "123") {
  ... on MintAccount {
    supply
  }
}
```

…and if that requires that we ask the RPC to parse the data using the `jsonParsed` mode, so be it.

To be pedantic, if we detect the presence of subselections in the query that we can't possibly fulfil except by fetching the thing in `jsonParsed` mode, then we should fetch the account in `jsonParsed` mode. Otherwise, we shouldn't.

If I _really_ want the underlying buffer, then we can offer this:

```graphql
account(address: "123") {
  data(encoding: BASE64)
}
```

Then we fetch it with the base64 encoding.

What if someone does this _totally legal query_?

```graphql
account(address: "123") {
  compressedBase64Data: data(encoding: BASE64_ZSTD)
  base64Data: data(encoding: BASE64_ZSTD)
  ... on MintAccount {
    supply
  }
}
```

Yep. We have to make that underlying fetch _three times_ with the current JSON RPC API.

Except when we rewrite the RPC, we can make this more efficient, and get all of the data in one shot.